integ-test/src/test/java/org/opensearch/sql/ppl/DateTimeFunctionIT.java

```diff
@@ -252,143 +234,38 @@ public void testDateAdd() throws IOException {
             rows("2019-08-11 00:00:00"));
   }
 
-  @Test
-  public void testDateTime() throws IOException {
```

Comment:
Please, revert. Those are tests for `datetime` function which is not removed.

Reply:
My mistake for removing these tests, I have added them back in in https://github.com/Bit-Quill/opensearch-project-sql/commit/f6edc6be62abbe3a3974665e8d51c515584234e0